R/pkg/R/column.R

```diff
@@ -151,6 +151,40 @@ setMethod("substr", signature(x = "Column"),
             column(jc)
           })
 
+#' startsWith
+#'
+#' Determines if entries of x start with string (entries of) prefix respectively,
+#' where strings are recycled to common lengths.
+#'
+#' @rdname startsWith
+#' @name startsWith
+#' @family colum_func
+#'
+#' @param vector of character string whose “starts” are considered
```

Comment:
`@param` should be once per parameter. I think we should leave out `@param` on the next line since it is describing the same parameter.
Also we should name the parameter first at the beginning of the text right after thr `@param` tag, see L146 for an example.
Finally, add `@param` and description for `x`